### ISSUE TITLE
Accessibility (screen-reader) bug: Name of the "Image" is not correct i.e. "MainInstructionIcon" instead of "Warning Icon".

### ISSUE THREAD

**Issue by Benjin** [electron/electron#19784]
Upstream via https://github.com/microsoft/azuredatastudio/issues/6666, then https://github.com/microsoft/vscode/issues/79072.  More details in the original.

TL;DR:

When producing a warning dialog box via `vscode.window.showWarningMessage()`, the accessible name of the image is "MainInstructionIcon", when it should be something like "Warning Icon" for screen-readers.  @isidorn says this is coming from Electron.

![image](https://user-images.githubusercontent.com/1609827/63116822-63d90780-bf4f-11e9-861a-2644ad35433a.png)

**Comment by ckerr** [comment]
As @isidorn [pointed out]() when referring you here, please create a small electron app which triggers a native dialog with a warning? Electron doesn't have `vscode.window.showWarningMessage()` in its API so a standalone repro would be preferable.

**Comment by Benjin** [comment]
Oh, sorry; I meant to update that to reference `dialog.showMessageBoxSync()` instead.  In any case, here's the quickstart app with that call added, and it repros there: https://github.com/Benjin/ElectronAccessibilityBugDemo

If you install https://accessibilityinsights.io/, then mouse-over the dialog that pops up when you run the modified quickstart app, you'll see the same thing as the screenshot.

**Comment by ckerr** [comment]
https://github.com/Benjin/electron-quick-start

**Comment by Benjin** [comment]
Yeah, I'm an idiot.  Link updated.

**Comment by isidorn** [comment]
@Benjin thanks for creating an issue here
fyi @deepak1556 since this feels like potential low hanging fruit to me

**Comment by miniak** [comment]
`dialog.showMessageBoxSync()` calls the `TaskDialogIndirect` API on Windows. Seems like the issue is on Windows side.
